Automattic/wp-calypso
# Removed unused Redux actions for email forwards

#### Proposed Changes

This is the 3rd PR in the process of migrating from Redux actions to react-query mutations in the email forwards management section.

This PR removes the Redux actions that have now been replaced by react-query mutations.

#### Testing Instructions



The removed actions are no longer in use, but to verify that the email forwards management still works as expected, we can follow the same steps as in #64650.

1. Have a domain without any email solution
2. Navigate to `/email/:domain/manage/:site_slug`
3. Click the "Start with Email Forwarding" link at the bottom of the page
4. Add a new email forward
5. Ensure that you are immediately taken to `/email/:domain/manage/:site_slug`
6. Ensure that you immediately see the newly added email forward on that page
7. Ensure that after a second or two, the newly added email forward displays a "Verification needed" warning
8. Click the "Resend verification email" button
9. Ensure that the destination email address receives another "Verify your email forwarding address" email
10. Remove the email forward
11. Ensure that it immediately disappears
12. Ensure that you are automatically taken to the "Pick an email solution" view after a few seconds




Related to #64650 and #64569
